addons/website_slides/models/slides.py

```diff
@@ -314,7 +314,8 @@ def _get_embed_code(self):
     def _website_url(self, name, arg):
         res = super(Slide, self)._website_url(name, arg)
         base_url = self.env['ir.config_parameter'].get_param('web.base.url')
-        res.update({(slide.id, '%s/slides/slide/%s' % (base_url, slug(slide))) for slide in self})
+        website_links = self.env['website.links']
+        res.update({(slide.id, website_links.sudo().create({'url': '%s/slides/slide/%s' % (base_url, slug(slide))}).short_url) for slide in self})
```

Comment:
as per channel settings (visibility) user can create/write slide. so we can not give create access rights of website_likes to those particular groups